sentry/src/main/java/io/sentry/protocol/DebugImage.java

```diff
@@ -75,6 +75,15 @@ public void setImageSize(Long imageSize) {
     this.imageSize = imageSize;
   }
 
+  /**
+   * for sentry-native compat.
```

Comment:
`l`: Does `sentry-java` actually know about `sentry-native`? If no, maybe we should change to comment here.

Reply:
no, it doesn't, but the reason to be added is `sentry-native`, I've added a more generic comment